compass/src/components/Api/ApiDetails/test/ApiDetails.test.js

```diff
@@ -1,46 +0,0 @@
-import React from 'react';
```

Comment:
Why was this test file deleted?

Reply:
It contains only commented out tests which, if I uncomment, throw errors.

Reply:
There is a comment as to why they are commented out:
`importing anything from ApiDetails causes Jest error because of generic-documentation component. So, no tests for now :(`
Seems like something which we could try to fix at some point in time, but removing this file would lose all these tests going forward, we'd forget they existed, when we are more free enough to look into it.

Reply:
Alright then - reverting. ◀️